src/Test.sol

```diff
@@ -139,6 +139,256 @@ abstract contract Test is DSTest {
             addr := create(0, add(bytecode, 0x20), mload(bytecode))
         }
     }
+
+    /*//////////////////////////////////////////////////////////////////////////
+                                    APPROX EQUAL
+    //////////////////////////////////////////////////////////////////////////*/
+
+    function assertApproxEq(
+        uint256 a,
+        uint256 b,
+        uint256 maxDelta
+    ) internal virtual {
+        uint256 delta = a > b ? a - b : b - a;
+
+        if (delta > maxDelta) {
+            emit log            ("Error: a ~= b not satisfied [uint]");
+            emit log_named_uint ("  Expected", b);
+            emit log_named_uint ("    Actual", a);
+            emit log_named_uint (" Max Delta", maxDelta);
+            emit log_named_uint ("     Delta", delta);
+            fail();
+        }
+    }
+
+    function assertApproxEq(
+        uint256 a,
+        uint256 b,
+        uint256 maxDelta,
+        string memory err
+    ) internal virtual {
+        uint256 delta = a > b ? a - b : b - a;
+
+        if (delta > maxDelta) {
+            emit log_named_string   ("Error", err);
+            emit log_named_uint     ("  Expected", b);
+            emit log_named_uint     ("    Actual", a);
+            emit log_named_uint     (" Max Delta", maxDelta);
+            emit log_named_uint     ("     Delta", delta);
+            fail();
+        }
+    }
+    function assertApproxEq(
+        int256 a,
+        int256 b,
+        int256 maxDelta
+    ) internal virtual {
+        int256 delta = a > b ? a - b : b - a;
+
+        if (delta > maxDelta) {
+            emit log            ("Error: a ~= b not satisfied [int]");
+            emit log_named_int  ("  Expected", b);
+            emit log_named_int  ("    Actual", a);
+            emit log_named_int  (" Max Delta", maxDelta);
+            emit log_named_int  ("     Delta", delta);
+            fail();
+        }
+    }
+
+    function assertApproxEq(
```

Comment:
Thoughts on naming the methods `assertApproxEqAbs` and `assertApproxEqRel`?  A bit different from solmate's naming but more clear IMO

Reply:
I prefer this, have made the change against the PR. Can unwind if people disagree